metrics/BUILD

```diff
@@ -13,17 +13,38 @@ filegroup(
     tags = ["automanaged"],
 )
 
+config_setting(  
```

Comment:
is it possible to put these in the `WORKSPACE` instead and then use the correct URL depending on the config setting?

Reply:
These suggest not but I'll verify:
https://github.com/bazelbuild/bazel/issues/1482
https://github.com/bazelbuild/bazel/issues/1485

Reply:
Yeah I get an error containing `config_setting cannot be in the WORKSPACE file`
At least the jq OSX binary is only 635kb.

Reply:
yeah, it looks like the only way to do this is to create a `repository_rule`, which is unfortunate.

that's probably generalizable and reusable, though I don't know if anyone has done that yet - seems like everyone rolls their own.